covariance_matrix_for_control_input_q: [1, 0, .032, 0,
                                        0, 1, .0, .032, 
                                        .08, 0.0, 1, 0,
                                        0, 0, 0, 1 ]
covariance_matrix_for_control_input_r: [5000, 0, .0, 0,
                                        0, 5000, .0, .0,
                                        .0, 0, 5000, 0,
                                        0, 0, 0, 5000]
covariance_matrix_for_yaw_angle_q: [1]
covariance_matrix_for_yaw_angle_r: [5000]
covariance_matrix_for_trajectory_tracker_q: [1, .0, .0, .0,
                                            .0, 1, .0, .0, 
                                            .0, .0, 1, .0,
                                            .0, .0, .0, 1 ]
covariance_matrix_for_trajectory_tracker_r:  [1, .0, .0, .0,
                                              .0, 1, .0, .0, 
                                              .0, .0, 1, .0,
                                              .0, .0, .0, 0.05 ]
covariance_matrix_for_position_q: [1, 0, .032, 0,
                                        0, 1, .0, .032, 
                                        .08, 0.0, 1, 0,
                                        0, 0, 0, 1 ]
covariance_matrix_for_position_r: [50, 0, .0, 0,
                                        0, 50, .0, .0,
                                        .0, 0, 50, 0,
                                        0,0,0, 50]
covariance_matrix_for_nmpc_q: [1, 0, .032, 0,
                                        0, 1, .0, .032, 
                                        .08, 0.0, 1, 0,
                                        0, 0, 0, 1 ]
covariance_matrix_for_nmpc_r: [200, 0, .0, 0,
                                        0, 200, .0, .0,
                                        .0, 0, 200, 0,
                                        0,0,0, 200]
